File: typescript-client-sdk/src/sdk/models/operations/deletecustomerscustomerid.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Type } from "class-transformer";


export class DeleteCustomersCustomerIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=customer_id" })
  customerId: string;
}

export class DeleteCustomersCustomerIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: DeleteCustomersCustomerIdPathParams;
}

export class DeleteCustomersCustomerIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: go-client-sdk/pkg/models/operations/deletecustomerscustomerid.go
package operations

type DeleteCustomersCustomerIDPathParams struct {
	CustomerID string `pathParam:"style=simple,explode=false,name=customer_id"`
}

type DeleteCustomersCustomerIDRequest struct {
	PathParams DeleteCustomersCustomerIDPathParams
}

type DeleteCustomersCustomerIDResponse struct {
	ContentType string
	StatusCode  int64
}
